src/serialization/compact/SchemaService.ts

```diff
@@ -106,4 +129,39 @@ export class SchemaService {
         const invocation = new Invocation(this.getInvocationService(), message);
         return this.getInvocationService().invokeUrgent(invocation).then(() => {});
     }
+
+    async replicateSchemaInCluster(schema: Schema): Promise<boolean> {
+        let clientMessage = ClientSendSchemaCodec.encodeRequest(schema);
+        outer:
+        for (let index = 0; index < this.maxPutRetryCount; index++) {
+            const invocation = new Invocation(this.getInvocationService(), clientMessage);
+            const response = await this.getInvocationService().invoke(invocation);
+            const replicatedMemberUuids = UuidUtil.convertUUIDSetToStringSet(ClientSendSchemaCodec.decodeResponse(response));
```

Comment:
Why do we convert UUID set to a string set?

Reply:
Because I could not use `Set.has` for objects.  And `ClientSendSchemaCodec.decodeResponse` return `Set<UUID>`.
Because of that I convert from it to `Set<string>`.

Reply:
You can use objects. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/has#using_the_has_method

Reply:
Turned out that we can't because SameValue semantics is used and the same object should be passed to the has() method, which is not possible